### PR TITLE
#5 Added Session-Cookie Code

### PR DESCRIPTION
**Pull request by AJV009**
This is the session code.
Except for the commented queries everything is correct as far as my assumptions go.
Hopefully after the DB part gets implemented I can test this thoroughly.

Made following changes:
- Edited README.md in the resources/src dir
- Added sessionCookie.php where all the session code exist
- Added redirect.php where all the redirection code exist (redirection code was not discussed earlier because I was late to remember about it and we are already delaying and out of time)

### CODE REVIEW

**Comment by AJV009** [comment]
@JayKandari  @guptahemant  take a look

**Comment by guptahemant** [comment]
@AJV009 squash the commits and add issue number in front of them please.

File: resources/src/sessionCookie.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?php
+include_once 'resources/template/redirect.php';
```
Comment (guptahemant) [inline resources/src/sessionCookie.php:2]: Try using autoloader, use statement and namespaces here.

File: resources/src/sessionCookie.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+class SessionCookie
+{
+    public $redirect = '';
+    public function __construct()
+    {
+        $this->redirect = new Redirect;
+        session_start();
+        // error_reporting(0);
+        ini_set('display_errors', 1);
+        ini_set('display_startup_errors', 1);
```
Comment (guptahemant) [inline resources/src/sessionCookie.php:12]: Try moving these to config file instead.

File: resources/template/redirect.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?php
+class Redirect
```
Comment (guptahemant) [inline resources/template/redirect.php:2]: Add a namespace.

**Review by guptahemant — CHANGES_REQUESTED** [review]

**Comment by AJV009** [comment]
You were reviewing?
I thought I messed up, so I created a copy and deleted the whole messed up repo
I mistakenly run some wrong commands

**Comment by guptahemant** [comment]
No worries, Try following the points i mentioned above in the next PR.

File: resources/src/sessionCookie.php (hunk 1 above)
Comment (AJV009) [reply resources/src/sessionCookie.php:2]: @guptahemant is using auto-loader all about
just adding something like ->
```php
spl_autoload_register('Redirect');

class Redirect
{
    public function jsalert($msg)
    { ...
```
Should we use composer overload?

Comment (AJV009) [reply resources/src/sessionCookie.php:2]: @guptahemant what do you exactly mean by statement here?

**Pull request closed without merging**